javascript/builtins/BigInt.json

```
@@ -269,6 +284,17 @@
                 "chrome_android": {
                   "version_added": "76"
                 },
+                "deno": [
+                  {
+                    "version_added": "1.8"
+                  },
+                  {
+                    "version_added": "1.0",
+                    "version_removed": "1.8"
```

This is my fault. Sorry!

```suggestion
                    "version_removed": "1.8",
```